pkg/logs/config/config.go

```diff
@@ -19,6 +19,15 @@ import (
 // ContainerCollectAll is the name of the docker integration that collect logs from all containers
 const ContainerCollectAll = "container_collect_all"
 
+const logsAgentPrefix = "agent-intake.logs."
+
+var logsEndpoints = map[string]int{
+	"agent-intake.logs.datadoghq.com": 10516,
+	"agent-intake.logs.datadoghq.eu":  443,
+	"agent-intake.logs.datad0g.com":   10516,
+	"agent-intake.logs.datad0g.eu":    443,
```

Comment:
I am not sure we want to expose staging endpoints to the world ..

Reply:
They are already discoverable (see https://github.com/DataDog/datadog-agent/blob/63dfe0f4b0205d2ecd39103665fefcfa74106e5a/pkg/config/config.go#L347) and several other. And the world doesn't have an API key